README.md

```diff
@@ -7,7 +7,7 @@ The purpose of this repo is to separate the concern of keyfile management from t
 # Installation
 This package can be installed from [PyPi.org](https://pypi.org/project/bittensor-wallet/):
 ```bash
-pip install bittensor-wallet==0.0.0
+pip install bittensor-wallet==0.0.1
```

Comment:
not pointing it out in terns of correctness but i'm curious about how we are going to proceed for packages naming. I think it would be nice to select a homogeneus naming strategy

we have btcli and now we are creating this bittensor-wallet, shouldn't we homogeneize this names? 
E.G having one of the following groups of names:
- btcli, btwallet
- bittensor-cli, bittensor-wallet

Waht do you think about this?